### PR TITLE
Added Teampopcorn's first blog post and photos

### PR DESCRIPTION
**Pull request by sleepypioneer**

### CODE REVIEW

**Comment by sleepypioneer** [comment]
@anasofiapinho I have failed checks for the deploy preview. Its because I dont have ruby set up on my home machine, is it ok without this check? I will set it up now incase it's not or can commit again from the coaching company in two hours.

File: blog/_posts/2018-07-09-welcome-to-team-popcorn.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Meet Team Popcorn
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/2018-07-09-welcome-to-team-popcorn.md
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-welcome-to-team-popcorn.md]: Hi @sleepypioneer! Thanks for the PR. I just have minor comments. You don't need markdown file extension (.md) for 'permalink'.

File: blog/_posts/2018-07-09-welcome-to-team-popcorn.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+current: blog
+author: Arati and Jessica
+twitter: TeamPopcornBer
+categories: student-posts
```
Comment (v-thakkar) [inline blog/_posts/2018-07-09-welcome-to-team-popcorn.md]: Just to maintain consistency across files and format for the posts, can you just change how you have declared categories parameter like below:

```
categories:
- student-posts
```

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by sleepypioneer** [comment]
Hi @nerdyvaishali we have made those changes :)

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**